repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -0,0 +1,58 @@
+= Attribute Mapping Policy Validation Filter
+
+include::../_includes/in-progress.adoc[]
```

Comment:
Since this is a brand new filter, pointing them back to the old Wiki is probably useless.